java/serve/src/main/java/io/ray/serve/common/Constants.java

```diff
@@ -15,7 +15,7 @@ public class Constants {
           1.0, 2.0, 5.0, 10.0, 20.0, 50.0, 100.0, 200.0, 500.0, 1000.0, 2000.0, 5000.0);
 
   /** Name of controller listen_for_change method. */
-  public static final String CONTROLLER_LISTEN_FOR_CHANGE_METHOD = "listen_for_change";
+  public static final String CONTROLLER_LISTEN_FOR_CHANGE_METHOD = "listen_for_change_xlang";
```

Comment:
```suggestion
  public static final String CONTROLLER_LISTEN_FOR_CHANGE_METHOD = "listen_for_change_java";
```
I think this name is more future proof, in case new languages become supported.

Reply:
Resolved. Rename 'xlang' to 'java'.
The reason why 'xlang' is used is that if c++ serve is available in the future, the interface on Python controller will be consistent with Java. But if c++ serve does not use python controller, this method name is not clear.

Reply:
Resolved.